server/src/main/java/org/apache/kafka/server/config/ZkConfigs.java

```diff
@@ -22,23 +22,15 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-import static org.apache.kafka.common.config.ConfigDef.Importance.HIGH;
 import static org.apache.kafka.common.config.ConfigDef.Importance.LOW;
 import static org.apache.kafka.common.config.ConfigDef.Importance.MEDIUM;
-import static org.apache.kafka.common.config.ConfigDef.Range.atLeast;
 import static org.apache.kafka.common.config.ConfigDef.Type.BOOLEAN;
-import static org.apache.kafka.common.config.ConfigDef.Type.INT;
 import static org.apache.kafka.common.config.ConfigDef.Type.LIST;
 import static org.apache.kafka.common.config.ConfigDef.Type.PASSWORD;
 import static org.apache.kafka.common.config.ConfigDef.Type.STRING;
 
 public final class ZkConfigs {
```

Comment:
open https://issues.apache.org/jira/browse/KAFKA-18631 to remove this config file